client/cli/lib/bumpVersion.ts

```diff
@@ -5,13 +5,14 @@ import { writeFileSync } from 'fs';
 
 interface Options {
   releaseType?: ReleaseType;
+  dir?: string;
 }
 
 export const releaseTypes = ['major', 'minor', 'patch', 'premajor', 'preminor', 'prepatch', 'prerelease'] as const;
 type ReleaseType = typeof releaseTypes[number];
 
-export default async ({ releaseType }: Options = {}, logger: any) => {
-  const packageJson = getPackageJson();
+export default async ({ releaseType, dir }: Options = {}, logger: any) => {
```

Comment:
we should have a default value for `dir` here I think...

Reply:
Only if you're not using workspaces

Reply:
also, the default is pwd, `getPackageJson` and `updatePackageJson` set it